src/v4/component/StoryKit/src/StoryListItem.tsx

```diff
@@ -562,56 +517,50 @@ export const StoryListItem = ({
-            item: content[current],
-          })
-        ) : (
-          <View style={styles.footer}>
-            {hasStoryImpressionPermission ? (
-              <TouchableOpacity
-                style={[
-                  styles.seenContainer,
-                  { backgroundColor: storyViewerBgColor },
-                ]}
-              >
-                <SvgXml xml={seenIcon()} width="25" height="25" />
-                <Text style={styles.seen}>{viewer}</Text>
-              </TouchableOpacity>
-            ) : (
-              <View style={styles.seenContainer} />
-            )}
-            <View style={styles.seenContainer}>
-              <TouchableOpacity
-                style={[
-                  styles.iconContainer,
-                  { backgroundColor: storyCommentBgColor },
-                ]}
-                onPress={onPressComment}
-              >
-                <SvgXml xml={storyCommentIcon()} width="25" height="25" />
-                <Text style={styles.seen}>{commentsCounts}</Text>
-              </TouchableOpacity>
-              <TouchableOpacity
-                style={[
-                  styles.iconContainer,
-                  {
-                    backgroundColor: storyReactionBgColor,
-                  },
-                ]}
-                onPress={onPressReaction}
-              >
-                <SvgXml
-                  xml={isLiked ? storyLikedIcon : storyLikeIcon}
-                  width="25"
-                  height="25"
-                />
-                <Text style={styles.seen}>{totalReaction}</Text>
-              </TouchableOpacity>
-            </View>
+
+        <View style={styles.footer}>
+          {hasStoryImpressionPermission ? (
+            <TouchableOpacity
+              style={[
+                styles.seenContainer,
+                { backgroundColor: storyViewerBgColor },
+              ]}
+            >
+              <SvgXml xml={seenIcon()} width="25" height="25" />
+              <Text style={styles.seen}>{viewer}</Text>
+            </TouchableOpacity>
+          ) : (
+            <View style={styles.seenContainer} />
+          )}
+          <View style={styles.seenContainer}>
+            <TouchableOpacity
+              style={[
+                styles.iconContainer,
+                { backgroundColor: storyCommentBgColor },
+              ]}
+              onPress={onPressComment}
+            >
+              <SvgXml xml={storyCommentIcon()} width="25" height="25" />
+              <Text style={styles.seen}>{commentsCounts}</Text>
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={[
+                styles.iconContainer,
+                {
+                  backgroundColor: storyReactionBgColor,
+                },
+              ]}
+              onPress={onPressReaction}
+            >
+              <SvgXml
+                xml={isLiked ? storyLikedIcon : storyLikeIcon}
+                width="25"
```

Comment:
I think width should be 20 or 24 for icon according to figma

Reply:
Sure.icons changed to 24